metar/Metar.py

```diff
@@ -443,7 +437,7 @@ def __init__(self, metarcode, month=None, year=None, utcdelta=None, strict=True)
                     if debug:
                         _report_match(handler, m.group())
                     handler(self, m.groupdict())
-                    code = code[m.end():]
+                    code = code[m.end() :]
```

Comment:
Some linter do this change?

Reply:
Yes, I saw black was used previously, so left it on to catch anything in these files. Happy to remove though if you want.

Reply:
Yes please and I think this PR is good to go then.  Thank you.